pkg/vere/io/http.c

```diff
@@ -1977,6 +2038,12 @@ _http_ef_http_server(u3_httd* htd_u,
   if ( c3y == u3r_sing_c("set-config", tag) ) {
     u3_http_ef_form(htd_u, u3k(dat));
   }
+  //  handles a cache notification
+  //
+  else if ( c3y == u3r_sing_c("grow", tag) ) {
+    // cache paths are /cache/(scot %u aeon)/(scot %t url)
+    u3h_put(htd_u->sax_p, u3t(u3t(dat)), dat);
```

Comment:
could be more explicit here:
```c
u3_noun pax = dat;
u3_noun url = u3t(u3t(pax));
```

Reply:
Be careful with refcounting when inserting into a hashtable.  The hashtable has different refcounting semantics for key vs. value.

Reply:
Ah, so leave the key alone, and `u3k` the value?

Reply:
```C
     /* u3h_put(): insert in hashtable.
      **
      ** `key` is RETAINED; `val` is transferred.
      */
        void
        u3h_put(u3p(u3h_root) har_p, u3_noun key, u3_noun val);
```

So yes, that sounds right.